apps/condo/domains/user/utils/testSchema/index.js

```diff
@@ -69,6 +72,28 @@ async function createTestUser (client, extraAttrs = {}, { raw = false } = {}) {
     return [result, attrs]
 }
 
+async function createTestUserExternalIdentity (client, extraAttrs = {}, { raw = false } = {}) {
```

Comment:
It looks like in new format we removed the raw argument

Reply:
agree